src/gwt/src/org/rstudio/studio/client/workbench/codesearch/ui/CodeSearchDialog.java

```diff
@@ -46,13 +50,18 @@ public CodeSearchDialog(Provider<CodeSearch> pCodeSearch)
    
    @Override
    protected Widget createMainWidget()
-   {     
-      VerticalPanel mainPanel = new VerticalPanel();
-      mainPanel.addStyleName(
+   {  
+      HorizontalPanel mainPanel = new HorizontalPanel();
+
+      VerticalPanel searchPanel = new VerticalPanel();
+      searchPanel.addStyleName(
          CodeSearchResources.INSTANCE.styles().codeSearchDialogMainWidget());
       codeSearch_ = pCodeSearch_.get();
       codeSearch_.setObserver(this);
-      mainPanel.add(codeSearch_.getSearchWidget());
+      searchPanel.add(codeSearch_.getSearchWidget());
+
+      mainPanel.add(searchPanel);
+      mainPanel.add(new HelpButton(new URL("https://support.rstudio.com/hc/en-us/articles/200710523-Navigating-Code-in-the-RStudio-IDE"), "Fuzzy finder help"));
```

Comment:
I don't know how the mapping between rstudioLinkName and the url is done, e.g. 

`new HelpButton("pdf_root_document", constants_.rootDocumentChooserTitle())` -> https://support.rstudio.com/hc/en-us/articles/200486298

Reply:
We maintain a set of redirects at https://github.com/rstudio/rstudio-web/blob/master/web/rstudio.org/site/links/.htaccess; you could add a redirect to this support article here.